packages/app/cypress/e2e/cypress-in-cypress-run-mode.cy.ts

```diff
@@ -62,6 +62,35 @@ describe('Cypress In Cypress - run mode', { viewportWidth: 1200 }, () => {
 
     cy.percySnapshot()
   })
+
+  it('hides reporter when NO_COMMAND_LOG is set in run mode', () => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.findBrowsers()
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer()
+    cy.withCtx(async (ctx, o) => {
+      const config = await ctx.project.getConfig()
+
+      o.sinon.stub(ctx.project, 'getConfig').resolves({
+        ...config,
+        env: {
+          ...config.env,
+          NO_COMMAND_LOG: 1,
+        },
+      })
+    })
+
+    cy.visitApp()
+    simulateRunModeInUI()
+    cy.contains('dom-content.spec').click()
+
+    cy.contains('http://localhost:4455/cypress/e2e/dom-content.html').should('be.visible')
+    cy.findByLabelText('Stats').should('not.exist')
+    cy.findByTestId('specs-list-panel').should('not.be.visible')
+    cy.findByTestId('reporter-panel').should('not.be.visible')
```

Comment:
Does this render the hidden div no content in it?

Reply:
Right, the panels are managed with a v-show, which means they are in the DOM with `display: none` on them until they appear. This is useful for the first panel since it has to appear instantly when opened, and preserve state when closed, but actually come to think of it, it's not important for the 2nd panel, since the command log cannot currently be toggled while the application is open, so we can change to a `v-if` here and lose div.